### PR TITLE
[windows] add a basic "Watchdog"

### PR DESCRIPTION
**Pull request by degemer**
- check memory consumption of all processes
- check whether the agent is blocked on a check or not

### CODE REVIEW

**Comment by degemer** [comment]
- Remove memory-based Watchdog as the previous experience with a memory-based Watchdog on Linux platforms was inconclusive.
- Remove automatic restart of the agent every day. It was introduced as a workaround of a memory leak on WS2003, and we are dropping support for WS2003.

File: win32/agent.py [hunk 1]
```diff
@@ -1,31 +1,41 @@
 # set up logging before importing any other components
+from config import get_version, initialize_logging; initialize_logging('collector')
```
Comment (remh) [inline win32/agent.py]: So I guess we should remove this line @degemer ?

Comment (degemer) [inline win32/agent.py]: Yes, forgot it. Thanks @remh !

**Comment by remh** [comment]
Thanks @degemer !

**Pull request merged**